io.seamly.seamly2d.metainfo.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id type="desktop">io.seamly.seamly2d</id>
+    <launchable type="desktop-id">io.seamly.seamly2d.desktop</launchable>
+    <name>Seamly2D</name>
+    <summary>Sewing pattern design software</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-3.0+</project_license>
+    <developer_name>Seamly2D project</developer_name>
+    <url type="homepage">https://seamly.io/</url>
+    <url type="bugtracker">https://github.com/FashionFreedom/Seamly2D/issues</url>
+    <url type="donation">https://www.paypal.com/donate?hosted_button_id=9NFZ87CECJN98</url>
+    <description>
+        <p>Seamly2D is pattern design software to create reusable, scalable, sharable custom-fit patterns using bespoke tailoring, haute couture, and historical pattern drafting techniques. Seamly2D saves time and improves fit by creating patterns that are reusable parametric CAD engineering documents.</p>
+        <p>Unlike other pattern design software, Seamly2D allows designers to have complete control of the patternmaking process. Each reusable, scalable pattern can read multi-size tables for boutique sizing, and read individual measurement files for custom-fit.</p>
+        <p>Seamly2D is open source software released under the GPLv3+ license. Seamly2D is available for Windows, MacOS, and Linux. Learn more about Seamly2D by joining our active, friendly community at https://forum.seamly.net</p>
+    </description>
+    <screenshots>
+        <screenshot type="default">
+            <image>https://raw.githubusercontent.com/FashionFreedom/Seamly2D/develop/share/img/Seamly2D_banner_700x200.png</image>
```

Comment:
the image should be pinned to a commit or tag, otherwise it will cause the build to break if it is removed from a branch.

Reply:
Thanks for the hint, done